assets/js/Components/Forms/ContrastForm.js

```diff
@@ -376,10 +315,10 @@ export default class ContrastForm extends React.Component {
     const element = Html.toElement(html)
 
     if (element.style.backgroundColor) {
-      return Contrast.rgb2hex(element.style.backgroundColor)
-    }
+      return Contrast.standardizeColor(element.style.backgroundColor)
```

Comment:
The name for this method has not been updated in Contrast.js

Reply:
It's now added to Contrast.js.